packages/uniforms-material/__tests__/SelectField.tsx

```diff
@@ -62,13 +62,11 @@ describe('@RTL - SelectField tests', () => {
     });
     const explicitProps = {
       fullWidth: false,
-      margin: 'normal' as const,
+      margin: 'dense' as const,
```

Comment:
I don't think that's a "fix" though? It makes the error go away but I believe `'normal'` should also be an acceptable prop value or is it not after all?

Reply:
yes, prop normal is normally acceptable, but mui has a hollow type definition in the `InputBase.d.ts` file, the definition says `margin? 'dense' | 'none'`. I fixed this by adding a prop margin in the interface of our SelectField